### ISSUE TITLE
Save button not working

### ISSUE THREAD

**Issue by laravignotto** [snizzo/lucrezia#5]
After loading and modifying a map, if I click on File > Save the program doesn't save the changes.

**Comment by laravignotto** [comment]
More specifically, when I save, in the terminal there's this error:

```
Traceback (most recent call last):
  File "~/lucrezia/src/editor/gui/QTTest.py", line 50, in saveScene
    m.save()
  File "~/lucrezia/src/editor/gui/MapExporter.py", line 26, in save
    self.bgImage = 'bgimage="'+pGrid.getBackgroundImage()+'"' if pGrid.getBackgroundImage() else ''
  File "~/lucrezia/src/grid/grid.py", line 188, in getBackgroundImage
    return self.bgImage
AttributeError: Grid instance has no attribute 'bgImage'
```

**Comment by snizzo** [comment]
complete fix: e3e68c26d5d200ff82bb8d8fc0d0c3c7cf3efd7b
